src/js/backend/permission.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const sudo = global.require('./node_modules/electron-sudo');
```

Comment:
This is for packaging. `electron-sudo` cannot be packed with other node modules, so I have to place it in a folder while other scripts are all compressed.